src/kubectlUtils.ts

```diff
@@ -0,0 +1,107 @@
+import * as vscode from "vscode";
+import { Kubectl } from "./kubectl";
+
+export type Cluster = {
+    name: string;
+    context: string;
+    user: string;
+    active: boolean;
+};
+
+export type Namespace = {
+    name: string;
+    active: boolean;
+};
+
+async function getKubeconfig(kubectl: Kubectl): Promise<any> {
+    const shellResult = await kubectl.invokeAsync("config view -o json");
+    if (shellResult.code !== 0) {
+        vscode.window.showErrorMessage(shellResult.stderr);

```

Comment:
Should we `await` this?  I guess it's not necessary.

Reply:
Looks like you do `await` message APIs in some other places in this file.

Reply:
nice

Reply:
i would prefer to remove `await` for showMessageBox in other places.